src/openvic-simulation/pop/Pop.cpp

```diff
@@ -206,7 +418,7 @@ bool PopManager::load_pop_type_file(
 		"issues", ZERO_OR_ONE, assign_variable_callback(issues_node),
 		"demote_migrant", ZERO_OR_ONE, expect_bool(assign_variable_callback(demote_migrant)),
 		"administrative_efficiency", ZERO_OR_ONE, expect_bool(assign_variable_callback(administrative_efficiency)),
-		"tax_eff", ZERO_OR_ONE, success_callback, // TODO - tax collection modifier
+		"tax_eff", ZERO_OR_ONE, expect_fixed_point(assign_variable_callback(tax_efficiency)),
 		"can_build", ZERO_OR_ONE, expect_bool(assign_variable_callback(can_build)),
 		"factory", ZERO_OR_ONE, expect_bool(assign_variable_callback(factory)),
```

Comment:
I think this is only used for the UI 'unemployed workers' icon.
Need to check. If so, please change the name in code.

Reply:
`factory` & `can_work_factory` are rather confusing terms. I'm looking into what they do and will update this comment.